src/translations/components/TranslationsAttributesPage/TranslationsAttributesPage.tsx

```diff
@@ -110,64 +115,68 @@ const TranslationsProductTypesPage: React.FC<TranslationsProductTypesPageProps>
         onSubmit={onSubmit}
       />
       <CardSpacer />
-      <TranslationFields
-        activeField={activeField}
-        disabled={disabled}
-        initialState={true}
-        title={intl.formatMessage(messages.values)}
-        fields={
-          data?.attribute?.choices?.edges?.map(
-            ({ node: attributeValue }, attributeValueIndex) => {
-              const isRichText =
-                attributeValue?.inputType === AttributeInputTypeEnum.RICH_TEXT;
-              const displayName = isRichText
-                ? intl.formatMessage({
-                    defaultMessage: "Text",
-                    description: "attribute richtext value"
-                  })
-                : intl.formatMessage(
-                    {
-                      defaultMessage: "Value {number}",
-                      description: "attribute values"
-                    },
-                    {
-                      number: attributeValueIndex + 1
-                    }
-                  );
+      {data &&
+        (withChoices ? null : (
+          <TranslationFields
+            activeField={activeField}
+            disabled={disabled}
+            initialState={true}
+            title={intl.formatMessage(messages.values)}
+            fields={
+              data?.attribute?.choices?.edges?.map(
+                ({ node: attributeValue }, attributeValueIndex) => {
+                  const isRichText =
+                    attributeValue?.inputType ===
+                    AttributeInputTypeEnum.RICH_TEXT;
+                  const displayName = isRichText
```

Comment:
same here, this condition can be skipped

Reply:
Fixed.